worker/dbaccessor/worker.go

```diff
@@ -692,6 +734,14 @@ func (w *dbWorker) shutdownDqlite(ctx context.Context, handover bool) {
 	w.dbApp = nil
 }
 
+func (w *dbWorker) requestAPIServerDetails() error {
+	_, err := w.cfg.Hub.Publish(apiserver.DetailsRequestTopic, apiserver.DetailsRequest{
```

Comment:
Do we care if the message got through? i.e. you can wait for the message to be published.

Reply:
This is unlikely to fail (if you compare with the x-controller pub/sub), because it's local-only.